refactor/urls.js

```diff
@@ -1,16 +1,25 @@
 // Refactor this code
 
 function getURL(dev = false, tasks = {}) {
-    let url = dev
-        ? `/tasks?status=ACTIVE&dev=true&size=20`
-        : '/tasks';
+  let url = "/tasks";
+  let params = new URLSearchParams();
```

Comment:
Why 'let'?

Reply:
I have changed it to var since when we again do re-initialization, we can persist it's value outside it's initialization